modules/running-your-dynamic-plug-in.adoc

```diff
@@ -15,7 +15,7 @@ You can run the plug-in using a local development environment. The OpenShift con
 
 .Procedure
 
-* Open two terminal windows in the local directory of your cloned repository.
+* Open two terminal windows in your local directory.
```

Comment:
```
Open two terminal windows in your local directory
```
it looks like we missed some context here, what repo shall we use?

Reply:
Would it be your local `openshift-console` directory or local `openshift-console/dynamic-plugin-sdk-webpack` directory?

Reply:
For Local development I would follow [these ](https://github.com/openshift/console/tree/master/dynamic-demo-plugin#local-development) that we are using for the dynamic-demo-plugin

Reply:
> For Local development I would follow [these ](https://github.com/openshift/console/tree/master/dynamic-demo-plugin#local-development) that we are using for the dynamic-demo-plugin

This is no longer the recommended approach... We should use the `yarn run start-console` command as was previously documented.